numpy/ma/tests/test_core.py

```diff
@@ -2982,6 +2982,30 @@ def test_trace(self):
                             X.trace() - sum(mXdiag.mask * X.diagonal(),
                                             axis=0))
 
+    def test_dot(self):
+        # Tests dot on MaskedArrays.
+        (x, X, XX, m, mx, mX, mXX, m2x, m2X, m2XX) = self.d
+        fx = mx.filled(0)
+        r = mx.dot(mx)
+        assert_almost_equal(r.filled(0), fx.dot(fx))
+        self.assertIs(r.mask, nomask)
```

Comment:
`assertIs` is not available in Python 2.6. Just use `assert_(r.mask is nomask)`.